ui/app/components/auth/page.js

```diff
@@ -40,14 +41,34 @@ export default class AuthPage extends Component {
 
   @tracked mfaAuthData;
   @tracked mfaErrors = '';
-  @tracked preselectedAuthType;
+  @tracked canceledMfaAuth = '';
 
   get cspError() {
     const isStandby = this.args.cluster.standby;
     const hasConnectionViolations = this.csp.connectionViolations.length;
     return isStandby && hasConnectionViolations ? CSP_ERROR : '';
   }
 
+  get authTabData() {
```

Comment:
This eventually included additional logic for handling the form view depending on the cluster's login customizations. I decided it should live here because it only needs to be computed once (based on the fetch mounts request in the route) and does not change depending on user interactions (which everything in `Auth::FormTemplate` does). 

It could also technically live in the route, but having it in the component makes it easier to test